src/event.c

```diff
@@ -34,6 +34,11 @@ enum {
 	MOT_RX, MOT_RY, MOT_RZ
 };
 
+enum {
+	KEY_RELEASE = 0,
```

Comment:
keeping with the surrounding nomenclature, I think these should be BN_PRESS/BN_RELEASE, instead of "key".

Reply:
yep